werf.yaml

```diff
@@ -1,21 +1,25 @@
 ---
 project: deckhouse
 configVersion: 1
-cleanup:
-  keepPolicies:
-  - references:
-      branch: /.*/
-      limit:
-        in: 720h
-  - references:
-      tag: /.*/
-      limit:
-        in: 720h
 gitWorktree:
   forceShallowClone: true
 ---
+# Base Images
+{{ $_ := set . "Images" (.Files.Get "candi/image_versions.yml" | fromYaml) }}
+  {{- range $k, $v := .Images }}
+    {{ $baseImagePath := (printf "%s%s" $.Images.REGISTRY_PATH (trimSuffix "/" $v)) }}
+    {{- if ne $k "REGISTRY_PATH" }}
```

Comment:
Why we remove .Images.REGISTRY_PATH ? It may useful in future.

Reply:
REGISTRY_PATH value is concatenated with other images from the file, except REGISTRY_PATH. It is still in the dictionary but without changes.

Reply:
sorry, I was inattentive. All is ok.